library/src/main/java/com/datatorrent/lib/fileaccess/FileAccess.java

```diff
@@ -151,7 +151,7 @@
      * @param value
      * @throws IOException
      */
-    void append(byte[] key, byte[] value) throws IOException;
+    void append(Slice key, Slice value) throws IOException;
```

Comment:
@chandnisingh 
Adding a method to an interface would also break backward compatibility for existing subclasses so declaring the old method deprecated and adding a new one actually does not help much except for the existing callers of the old method. The only caller of the old method in the entire Malhar is BucketsFileSystem. Granted there may be some code outside of Apex that will be affected by this change but I think it's not worth keeping around that method just because of that possibility.